src/components/Navbar.jsx

```diff
@@ -0,0 +1,139 @@
+import PropTypes from "prop-types";
+import { useEffect, useRef, useState } from "react";
+import { BsTranslate } from "react-icons/bs";
+import { RxHamburgerMenu } from "react-icons/rx";
+
+import logoSrc from "../assets/logo.svg";
+
+const Navbar = ({ className, sections, lngs }) => {
```

Comment:
nit: can lngs be languages? i think it's clearer and the word is not too long that it needs to be shorten

Reply:
this is to keep consistent with the convention used by `react-i18next`: https://react.i18next.com/getting-started